src/libs.ts

```diff
@@ -6,7 +6,9 @@ const E = process.env
  * Singleton wrapper for Audius Libs.
  * Initialized in the server start-up.
  */
+// @ts-ignore - missing types on AudiusLibs
```

Comment:
i think you can do `import type {AudiusLibs} from '@audius/sdk'`

Reply:
sozz